src/Filters/coefficients.jl

```diff
@@ -380,3 +380,19 @@ SecondOrderSections(f::FilterCoefficients) = SecondOrderSections(ZeroPoleGain(f)
 *(f1::Biquad, fs::Biquad...) = SecondOrderSections([f1, fs...], 1)
 *(f1::SecondOrderSections, f2::Biquad) = SecondOrderSections([f1.biquads; f2], f1.g)
 *(f1::Biquad, f2::SecondOrderSections) = SecondOrderSections([f1; f2.biquads], f2.g)
+inv(f::FilterCoefficients) = begin
+    filt_pr = PolynomialRatio(f)
+    return typeof(f)(PolynomialRatio(filt_pr.a, filt_pr.b))
+end
+^(f::FilterCoefficients, n::Integer) = begin
```

Comment:
Why not make it a multi-line function definition:

```julia
function ^(f::FilterCoefficients, n::Integer)
    ...
end
```

Reply:
If that's the preferred style I'll happily change it.